contrib/gcppubsub/config/300-gcppubsubsource.yaml

```diff
@@ -45,14 +45,22 @@ spec:
           properties:
             gcpCredsSecret:
               type: object
+              description: "Credential to use to poll the GCP PubSub Subscription. It is not used to create or delete the Subscription, only to poll it. The value of the secret entry must be a service account key in the JSON format (see https://cloud.google.com/iam/docs/creating-managing-service-account-keys)."
             googleCloudProject:
               type: string
+              description: "ID of the Google Cloud Project that the PubSub Topic exists in."
```

Comment:
I would add an example here, as historically GCP project name vs ID vs display name has been very confusing.

description: "ID of the Google Cloud Project that the PubSub Topic exists in. E.g. 'my-project-1234' rather than its display name, 'My Project' or its number '1234567890'."